src/supla/network/esp_web_server.cpp

```diff
@@ -77,7 +77,11 @@ void postHandler() {
       // with very long timeout. In order to prevent app from hanging, we
       // skip getHandler in such case.
       if (reboot != 1) {
-        getHandler();
+       //zamiana
+        //getHandler();
+       //na zapobiegająca ponownemu przesyłaniu formularza przy odświerzaniu
```

Comment:
usuń proszę te komentarze.
można dodać
// Sending redirect after POST, in order to get rid of sending another POST on page refresh in browser